second-edition/src/ch09-02-recoverable-errors-with-result.md

```diff
@@ -18,8 +18,8 @@ enum Result<T, E> {
 `T` и `E` параметры перечисления. `T`  - это тип, которые будет возвращён, при
 успехе, а `E` при ошибке.
 
-Let’s call a function that returns a `Result` value because the function could
-fail: opening a file, shown in Listing 9-2.
+Давайте вызовем функцию, которая возвращает значение Result из-за сбоя функции:
+открытие файла, как показано в примере 9-2.
```

Comment:
В текущей версии книги так:
> Let’s call a function that returns a `Result` value because the function could fail. In Listing 9-3 we try to open a file.

Предлагаю синхронизировать с ним и написать так:
> Давайте вызовем функцию, которая возвращает значение `Result`, потому что эта функция может потерпеть неудачу. В листинге 9-2 мы пытаемся открыть файл.